src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -57,7 +57,7 @@ const FeedModalItem = (props: FeedEventProps) => {
           <TopImage type={itemType} />
           <ModalPaymentStatus item={item} />
           <View style={styles.dateAndAmount}>
-            {item.id?.startsWith('0x') && !itemType.match(/native/) && (
+            {item.id?.startsWith('0x') && !item.data?.asset && (
```

Comment:
why aren't we opening explorer for native txs? and asset can be in the future any erc20 token

Reply:
we open.
because native txs have no user data (e.g. no avatar), the network icon for them displays as avatar. and it's clickable showing tx on explorer https://github.com/GoodDollar/GoodDAPP/pull/4015/commits/5fa76d7678a0924e2e96d1cd57af06e34230ec3f

Reply:
we don't use inbox here. everything as simple as possible. you could open an enhancement issue for this in Zenhub

Reply:
still the avatar of the sender should be displayed

Reply:
@sirpy please open a ticket for this

Reply:
@sirpy done https://github.com/GoodDollar/GoodDAPP/pull/4015/commits/b491b6efc19adb737206d7a51e8240cbc647c2a9